### ISSUE TITLE
itemWrapperClasses is not taken into account in the vue Row

### ISSUE THREAD

**Issue by ragingdave** [64robots/nova-fields#103]
Basically exactly what I said in the title the configurable itemWrapperClasses property while sent to the frontend isn't being utilized making it a dead configurable option.

To be a bit clearer, the itemWrapperClasses aren't being taken into account on the form page resulting in unusual formatting that I could fix by changing the itemWrapperClasses, if they were used:

![Screen Shot 2019-06-11 at 12 03 20 PM](https://user-images.githubusercontent.com/1168344/59288109-fe138b00-8c40-11e9-90d0-af24d2032f59.png)

**Comment by disitec** [comment]
Hi @ragingdave, thanks for the report.

That config option is being used in the detail field as you can see here
https://github.com/64robots/nova-fields/blob/master/resources/js/components/row/DetailField.vue#L26

Do you think it could be applied also in FormField somehere?

For FormField we are using `rowWrapperClasses` for formatting the wrapper. Is this enough for you?
https://github.com/64robots/nova-fields/blob/master/resources/js/components/row/FormField.vue#L20

Let me know if you miss some config somewhere else.

P.S. We are working, finally, in the docs (docs branch)

**Comment by ragingdave** [comment]
My confusion was that itemWrapperClasses is taken into account on the Detail Field but not the FormField and then really why would there be 2 configurations that apply to the same thing.....but not be the same thing. If itemWrapperClasses and rowWrapperClasses are the same thing (or at least affect the same thing) why are they separate in the first place?

I realized in writing this that I was applying the css too high via the Row itself and was able to apply the styling I needed via the Textarea in the Row itself so it worked out via wrapperClasses and fieldClasses on the Textarea, rowWrapperClasses would have been enough in this case.

**Comment by temp1029** [comment]
I'm having trouble controlling the width of columns for data rows. I think this is what I need (based on the default value indicated in the documentation compared to what I see in the markup), but from what I'm reading here the 'itemWrapperClasses' function doesn't affect the wrappers around the form fields.

Here is what I'm seeing, with the header row OK and the data rows smushed to the left:
![Screenshot from 2019-12-26 13-51-42](https://user-images.githubusercontent.com/4396942/71487183-064fa400-27e8-11ea-8eee-cae7b5edbe7f.png)

And here is the code generating this:
![Screenshot from 2019-12-26 14-00-26](https://user-images.githubusercontent.com/4396942/71487198-1b2c3780-27e8-11ea-8943-7ee389cabcf3.png)

I should note that the detail view is displaying exactly like I want (can provide a screenshot of this if it would help). Also, when I manipulate the HTML directly (firefox inspector) I am able to achieve the results I'm trying for, but I can't figure out how to apply them via 'classes' methods.

**Comment by disitec** [comment]
You can inspect this file
https://github.com/64robots/nova-fields/blob/master/src/Row.php
and compare it with the original classes you are changing in order to know what method you should use.
Hope it helps!

**Comment by temp1029** [comment]
I actually found the answer in a different file, using the process you suggested. I needed to call wrapperClasses on each field per the code in this file:

https://github.com/64robots/nova-fields/blob/7e6c7047af9f44ec6ce0ae76d5e7677610cd1bff/src/Configurable.php#L15

Still having some issues with getting the display right across the details and form but good enough for now. Thanks for pointing me in the right direction.
